ios/FluentUI.Demo/FluentUI.Demo/Demos/CommandBarDemoController.swift

```diff
@@ -75,6 +75,24 @@ class CommandBarDemoController: DemoController {
             }
         }
 
+        var accentImage: UIImage? {
+            switch self {
+            case .delete:
+                return UIImage(named: "delete24Filled")
+            default:
+                return nil
+            }
+        }
+
+        var accentImageTintColor: UIColor? {
+            switch self {
+            case .delete:
+                return Colors.communicationBlue
```

Comment:
can you make it primary color please? in fluent demo app we can easily change from blue to green branding color to accomodate various client sceanrios.